sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -381,6 +381,17 @@ public partial class BlobCorsRule
         public string ExposedHeaders { get { throw null; } set { } }
         public int MaxAgeInSeconds { get { throw null; } set { } }
     }
+    public partial class BlobDeleteOptions
+    {
+        public BlobDeleteOptions() { }
+        public Azure.Storage.Blobs.Models.BlobRequestConditions Conditions { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.BlobDeleteType? DeleteType { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.DeleteSnapshotsOption? SnapshotsOption { get { throw null; } set { } }
+    }
+    public enum BlobDeleteType
+    {
+        Permanent = 0,
+    }
```

Comment:
I'd rather make two options in `BlobDeleteType` instead of making `DeleteType` property optional. Otherwise why don't we use simple `bool`? (btw. I think `BlobDeleteType` has low risk of growing beyond two options, on the other hand it seems to enum on server side might be it's expected to grow)

Reply:
I suspect it will grow, since it is not a bool in the service.